src/praisonai-code/praisonai_code/cli/interactive/async_tui.py

```
@@ -1,6 +1,1866 @@
+            # running without the shared bridge when it is unavailable.
+            from praisonai_code._wrapper_bridge import import_wrapper_module
+            run_sync_or_offload = import_wrapper_module(
+                "praisonai._async_bridge"
+            ).run_sync_or_offload
+            run_sync_or_offload(
+                self._start_runtime(),
+                thread_name="praisonai-tui-runtime",
+            )
+            # Runtime status logged to debug file only (not shown in UI)
+            # Tools are available silently when runtime is ready
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)